pkg/testing/command/command.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2019-2021 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package command
+
+import (
+	"testing"
+
+	igrunner "github.com/inspektor-gadget/inspektor-gadget/pkg/testing/ig"
+	"github.com/stretchr/testify/require"
+)
+
+type Command struct {
+	// IG wrapper
+	igrunner.IG
```

Comment:
I think we can have this relation in the other way around: Command to be a generic way or running commands, and `igrunner` a wrapper around it focused on ig.

Something like https://github.com/inspektor-gadget/inspektor-gadget/commit/8f44104dc2b2835c05fdaea763aaaac9560f2f81